apps/cms/scripts/seed-dev.mjs

```diff
@@ -146,82 +116,70 @@ async function main() {
    * Create application and application countries
    */
   console.log('- Application')
-  const appId = '6048041f-2d72-4eb7-9a2c-3ab44aace8d5'
-  await updateEntityRecord(directus, 'application', '', {
-    id: appId,
-    currency: 'USD',
+  const application = await directus.singleton('application').update({
+    currency: 'USD'
   })
-  await createEntityRecords(directus, 'application_countries', {
-    id: 1,
-    application_id: appId,
-    countries_code: 'US',
+  await directus.items('application_countries').createOne({
+    application_id: application.id,
+    countries_code: 'US'
   })
 
   /**
    * Create homepage
    */
   console.log('- Homepage')
   const homepageFactory = Factory.build('homepage')
-  const homepage = await updateEntityRecord(
-    directus,
-    'homepage',
-    '',
-    homepageFactory
-  )
-
+  const homepage = (await directus.singleton('homepage').update(homepageFactory))
   /**
    * Create rarities.
    */
   console.log('- Rarities')
   const rarityFactories = Factory.buildList('rarity', numRarities)
-  const rarities = await createEntityRecords(
-    directus,
-    'rarities',
-    rarityFactories
-  )
+  const rarities = (await directus.items('rarities').createMany(rarityFactories)).data
 
   /**
    * Creates N collectibles with mixed rarities.
    */
   console.log('- NFT Files')
   let notableCount = 0
-  const collectibleFactories = await Promise.all(
-    Factory.buildList('collectible', numCollectibles).map(async (item, i) => {
-      const isNotable = notableCount < 8
-      if (isNotable) {
-        notableCount++
-      }
-      // Using rarityLikelihood will apply rarities to some of the items.
-      const rarity = rarities[i % rarityLikelihood]
-      item.rarity = rarity?.id || null // no rarity if null
-
-      const previewImage = await makeImage({
-        directus,
-        text: item.translations[0].title,
-        color: '#000000',
-        borderColor: rarity?.color || '#000000',
-        backgroundColor: randColor(),
-      })
-
-      item.preview_image = previewImage.id
+  const buildList = Factory.buildList('collectible', numCollectibles)
```

Comment:
I switched this to a pure synchronous for loop for two reasons

1. As `Promise.all`, the calls to `createAssetRecords` in `makeImage` queue up and then get called all at the same time. With larger multipliers this was bombarding the CMS with calls and causing service unavailable errors. By making it purely synchronous, it is much easier on the CMS
2. Additionally with larger multipliers, this process can hang for a while. I added a console log for each image being processed so you can have an idea that it's actually doing something and not hanging indefinitely